### PR TITLE
update possible (standard) denominations

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

**Comment by lontivero** [comment]
Not sure this is good. It is a detail that seems to belong to the technical documentation at best.

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -736,16 +736,17 @@ With Wasabi we are trying to do lower estimations, rather than higher ones.
 :::details
 ### What are the equal denominations created in one mixing round?
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md:737]:
```suggestion
### What are the equal denominations created in a coinjoin round?
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -734,18 +734,19 @@ With Wasabi we are trying to do lower estimations, rather than higher ones.
...
-This is the mathematically most efficient way to get anonymity set for any amount you desire.
-For example, with a 0.7 BTC input you would expect the following outputs: ~0.1, ~0.2, ~0.4 BTC.
-
-@[youtube](PKtxzSLPWFU)
-
-@[youtube](3Ezru07J674)
+### What are the equal denominations created in a coinjoin round?
+
+The standard denominations are:
+`5000, 6561, 8192, 10000, 13122, 16384, 19683, 20000, 32768, 39366, 50000, 59049, 65536, 100000, 118098,
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:740]:
```suggestion
`5000`, `6561`, `8192`, `10000`, `13122`, `16384`, `19683`, `20000`, `32768`, `39366`, `50000`, `59049`, `65536`, `100000`, `118098`, `131072`, `177147`, `200000`, `262144`, `354294`, `500000`, `524288`, `531441`, `1000000`, `1048576`, `1062882`, `1594323`, `2000000`, `2097152`, `3188646`, `4194304`, `4782969`, `5000000`, `8388608`, `9565938`, `10000000`, `14348907`, `16777216`, `20000000`, `28697814`, `33554432`, `43046721`, `50000000`, `67108864`, `86093442`, `100000000`, `129140163`, `134217728`, `200000000`, `258280326`, `268435456`, `387420489`, `500000000`, `536870912`, `774840978`, `1000000000`, `1073741824`, `1162261467`, `2000000000`, `2147483648`, `2324522934`, `3486784401`, `4294967296`, `5000000000`, `6973568802`, `8589934592`, `10000000000`, `10460353203`, `17179869184`, `20000000000`, `20920706406`, `31381059609`, `34359738368`, `50000000000`, `62762119218`, `68719476736`, `94143178827`, `100000000000`, `137438953472` sats.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -734,18 +734,19 @@ With Wasabi we are trying to do lower estimations, rather than higher ones.
...
+131072, 177147, 200000, 262144, 354294, 500000, 524288, 531441, 1000000, 1048576, 1062882, 1594323, 2000000,
+2097152, 3188646, 4194304, 4782969, 5000000, 8388608, 9565938, 10000000, 14348907, 16777216, 20000000,
+28697814, 33554432, 43046721, 50000000, 67108864, 86093442, 100000000, 129140163, 134217728, 200000000,
+258280326, 268435456, 387420489, 500000000, 536870912, 774840978, 1000000000, 1073741824, 1162261467,
+2000000000, 2147483648, 2324522934, 3486784401, 4294967296, 5000000000, 6973568802, 8589934592, 10000000000,
+10460353203, 17179869184, 20000000000, 20920706406, 31381059609, 34359738368, 50000000000, 62762119218,
+68719476736, 94143178827, 100000000000, 137438953472 sats`
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:747]:
```suggestion
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]
I tend to agree with Lucas on this, but the current answer is wrong so let's merge this.

**Pull request merged**